src/tests/test_graphvisitor.cc

```diff
@@ -32,12 +32,10 @@ using namespace votca::tools;
 
 class GraphVisitorTest : public GraphVisitor {
  private:
-  void addEdges_(const Graph&, int) override {
-    throw runtime_error("Undefined method.");
-  }
-  Edge getEdge_(const Graph&) override {
+  void addEdges_(const Graph&, long int) override {
     throw runtime_error("Undefined method.");
   }
+  Edge getEdge_() override { throw runtime_error("Undefined method."); }
```

Comment:
@JensWehner maybe `Edge getEdge_() = delete;`

Reply:
yeah for all of them